src/charm.py

```diff
@@ -398,7 +391,7 @@ def _on_install(self, event: InstallEvent) -> None:
 
         # Construct the mongod startup commandline args for systemd and reload the daemon.
         update_mongod_service(
-            auth=auth,
+            auth=True,
```

Comment:
IIRC the only scenario in which we started `mongod` service without auth, was when we were using legacy relations.

Would you mind updating this function to not using this parameter anymore?

Reply:
Done!